### DISCUSSION TITLE
All option in makeInputSelect() filter returns no data

### DISCUSSION THREAD

**Discussion by abhaypithadiya** [Power-Components/livewire-powergrid#757] [BUG]
While using the makeInputSelct() filter, It filters the data when selecting other options but when we change it to all the table comes out to be empty.

**Comment by abhaypithadiya** [comment]
I am not sure if this is the write way but I published the views and added `value =" "` in select.blade.php and it works.

**Reply by M4x28** [reply]
Show me the solution, bc I'm having the same problem

**Reply by abhaypithadiya** [reply]
Hey! 

1) Publish the Powergrid view file using:

`php artisan vendor:publish --tag=livewire-powergrid-views`

2) Go to resources\views\vendor\livewire-powergrid\components\frameworks\bootstrap5\filters\select.blade.php, On line 16 add value="" in the option tag.

**Reply by M4x28** [reply]
Unfortunately it doesn't work for me, I have the same issue. I think is something with loading data

**Reply by abhaypithadiya** [reply]
Maybe, is the data loading by joining two tables? If so make sure that the id you are joining with should exists in the other table.
